### PR TITLE
Change doc.featurePolicy to .permissionsPolicy

### PR DESCRIPTION
**Pull request by marcoscaceres**
Update to reflect the API rename.

### CODE REVIEW

File: permissions-policy/resources/permissions-policy.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Feature test to avoid timeouts
 function assert_permissions_policy_supported() {
-  assert_not_equals(document.featurePolicy, undefined,
+  assert_not_equals(document.permissionsPolicy, undefined,
                     'permissions policy is supported');
```
Comment (marcoscaceres) [inline permissions-policy/resources/permissions-policy.js:4]:
```suggestion
  assert_not_equals(document.permissionsPolicy, undefined,
                    'permissions policy is supported');
  assert_equals(document.featurePolicy, undefined,
                    'permissions policy is supported');
```

File: permissions-policy/resources/permissions-policy.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // Feature test to avoid timeouts
 function assert_permissions_policy_supported() {
-  assert_not_equals(document.featurePolicy, undefined,
+  assert_not_equals(document.permissionsPolicy, undefined,
+                    'permissions policy is supported');
+  assert_equals(document.featurePolicy, undefined,
                     'permissions policy is supported');
```
Comment (marcoscaceres) [inline permissions-policy/resources/permissions-policy.js:6]:
```suggestion
  assert_equals(document.featurePolicy, undefined,
                    'feature policy should no longer be be supported');
```

File: permissions-policy/resources/permissions-policy.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
 // Feature test to avoid timeouts
 function assert_permissions_policy_supported() {
-  assert_not_equals(document.featurePolicy, undefined,
+  assert_not_equals(document.permissionsPolicy, undefined,
                     'permissions policy is supported');
+  assert_equals(document.featurePolicy, undefined,
+                    'feature policy should no longer be be supported');
```
Comment (marcoscaceres) [inline permissions-policy/resources/permissions-policy.js:6]:
```suggestion
  assert_equals(document.featurePolicy, undefined,
                    'feature policy should no longer be supported');
```

File: permissions-policy/resources/permissions-policy.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
 // Feature test to avoid timeouts
 function assert_permissions_policy_supported() {
-  assert_not_equals(document.featurePolicy, undefined,
+  assert_not_equals(document.permissionsPolicy, undefined,
```
Comment (marcoscaceres) [inline permissions-policy/resources/permissions-policy.js:3]:
```suggestion
  assert_equals(document.permissionsPolicy, undefined,
```

**Comment by clelland** [comment]
I'm all for deprecating the old name, but I'd prefer to do it in a way that doesn't also cause all of the functionality tests to appear broken in Chromium -- could we add a single WPT that asserts the non-existence of `document.featurePolicy`, rather than have its presence cause the remaining tests to be skipped?

(I'd love to get rid of it, but [this graph](https://chromestatus.com/metrics/feature/timeline/popularity/2511) suggests that it's going to take a while)

File: permissions-policy/resources/permissions-policy.js [hunk 5]
```diff
@@ -1,7 +1,8 @@
...
```
Comment (marcoscaceres) [inline permissions-policy/resources/permissions-policy.js:3]:
```suggestion
  assert_not_equals(HTMLIFrameElement.prototype.allow, undefined,
```

File: permissions-policy/resources/permissions-policy.js [hunk 6]
```diff
@@ -1,7 +1,8 @@
...
                     'permissions policy is supported');
+  assert_equals(document.featurePolicy, undefined, 'feature policy should no longer be supported');
```
Comment (marcoscaceres) [inline permissions-policy/resources/permissions-policy.js:5]:
```suggestion
```

File: permissions-policy/resources/permissions-policy.js (hunk 5 above)
Comment (clelland) [reply permissions-policy/resources/permissions-policy.js:3]: Thanks! LGTM

**Review by clelland — APPROVED** [review]

File: permissions-policy/resources/permissions-policy.js [hunk 7]
```diff
@@ -1,6 +1,6 @@
 // Feature test to avoid timeouts
 function assert_permissions_policy_supported() {
-  assert_not_equals(document.featurePolicy, undefined,
+  assert_true("allow" in HTMLIFrameElement.prototype,
                     'permissions policy is supported');
```
Comment (marcoscaceres) [inline permissions-policy/resources/permissions-policy.js:4]:
```suggestion
  assert_true("allow" in HTMLIFrameElement.prototype, 'permissions policy is supported');
```

**Pull request merged**